submissions/idiagheOsaigbovo/CssTask/login/style.css

```diff
@@ -0,0 +1,83 @@
+body
+{
+    margin: 25;
+    padding: 0;
+    font-family: sans-serif;
+    background-image: url(../../pictures/background-2.jpg);
+    background-size: cover;
+}
+.box
+{
+    margin: 200px auto;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50,-50);
+    width: 400px;
+    padding: 40px;
+    padding-top: 50px;
+    background: rgba(0,0,0,.8);
+    box-sizing: border-box;
+    box-shadow: 0 15px 25px rgba(0,0,0,.5);
+    border-radius: 10px;
+   
+
+}
+.box h2
```

Comment:
Please place the curly bracket on the same line as the selector. Same applies to all selectors in this css file.